### PR TITLE
pages/common: fix typos in placeholders

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Ref. #11850

Although this still will have a non-placeholder bracket it is much better than not being shown itself.

image

### CODE REVIEW

**Review by vitorhcl — APPROVED** [review]
Sorry for that, didn't pay enough attention when doing those changes

**Comment by acuteenvy** [comment]
> Although this still will have a non-placeholder bracket

I've already mentioned this [here](https://github.com/tldr-pages/tldr/pull/10730#issuecomment-1726682739) - `{{{something}}}` is undefined by our client specification.

https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#page-structure
> the non-standard `{{` and `}}` placeholder syntax surrounds values in an example that users may edit.

This is ambiguous. The "value" in this case can be:
- `{something}` (outer braces mark the placeholder)
- `something`   (inner braces mark the placeholder)
- `{something`  (outer at the beginning, inner at the end)
- `something}`  (inner at the beginning, outer at the end)

This is possible to fix, but we have to clearly define that the outer braces mark the placeholder.

**Review by blueskyson — APPROVED** [review]

**Comment by kbdharun** [comment]
> This is possible to fix, but we have to clearly define that the outer braces mark the placeholder.

Yeah, we can add something like this to the client specification in future.

File: pages/common/git-stash.md
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/git-stash.md:1]: Merged the PRs branch with main, it should be addressed now.

**Pull request merged**